docs/builds.md

```diff
@@ -1,76 +1,31 @@
-# GitHub Actions - Builds
+# Showing GitHub Actions builds in Jira
 
-GitHub for Jira supports builds via [GitHub Actions workflow syntax](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions).
-To set this up, you will need to create a .github folder at the root of a given repository and then make a child directory
-called workflows. Inside of .github/workflows you will need to create a build.yml file. This is where you will specify the workflow for your builds.
+The GitHub for Jira app automatically processes "workflow runs" from any [GitHub Action workflows](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions) you have defined in your GitHub repositories and shows them in the "Builds" tab of the development info panel of a Jira issue:
 
-Following is an example of a build.yml file:
```

Comment:
Removed the example, because it's pretty much exactly the example of the GitHub Actions docs and we should just link there (it's not our job to explain how GitHub workflows work)